tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -1992,6 +1992,10 @@ int CorProfiler::RegisterIastAspects(WCHAR** aspects, int aspectsLength)
     return 0;
 }
 
+void CorProfiler::GetIastMetrics(int* instrumentedSources, int* instrumentedPropagations, int* instrumentedSinks)
+{
+    _dataflow->GetIastMetrics(instrumentedSources, instrumentedPropagations, instrumentedSinks);
```

Comment:
This could crash if `_dataflow` is `null`

Reply:
Fixed!